simtools/configuration/commandline_parser.py

```diff
@@ -12,12 +12,18 @@ class CommandLineParser(argparse.ArgumentParser):
 
     Command line arguments should be given in snake_case, e.g. `input_meta`.
 
-    Methods
+    Atributes
```

Comment:
Remember that numpy requires the underlines to be exactly the same length as the title
```
Attributes
----------
```